package.json

```
@@ -22,11 +14,15 @@
     "vaults"
   ],
   "author": "Morpho Labs",
```

Not sure that it is useful
```suggestion
  "author": "Morpho Labs <security@morpho.xyz>",
```